docs/content/direct/observability.md

```diff
@@ -0,0 +1,63 @@
+# Observability in KubeStellar
+
+KubeStellar provides endpoints and integrations for observability and monitoring. This page describes the available observability features, how to access them, and how to use them in a typical deployment.
+
+## Metrics Endpoints
+
+KubeStellar controllers expose Prometheus-compatible metrics endpoints. These endpoints respond to HTTP requests for metrics and can be queried by any monitoring system; KubeStellar does not mandate how metrics are collected or scraped.
+
+### Enabling Metrics
+
+KubeStellar controllers expose metrics at the `/metrics` endpoint by default. You may use any monitoring stack you prefer to collect these metrics.
+
+For an example of collecting both metrics and debug endpoint data using Prometheus, see the [monitoring/README.md](https://github.com/kubestellar/kubestellar/tree/main/monitoring/README.md). This is just one possible approach; KubeStellar does not require or mandate any specific monitoring tool or method.
+
+### Metrics Endpoint Table
+
+| Controller                      | Protocol | Port  | Path      | AuthN/AuthZ | Notes |
```

Comment:
The port listed here is only the _default_. In the core Helm chart, which is currently the only supported way to set up, the ports to use are configured in the "values". The reader should be told this.